mandatory/1-fix-functions.js

```diff
@@ -21,7 +21,7 @@ function mood() {
 }
 
 function greaterThan10(num) {
```

Comment:
Personally this is where I'd use the ternary operator like: `return num > 10 ? "num is greater than 10" : "num is not big enough"` but there is nothing wrong with what you've written!

Reply:
yea, it's very practical, I updated mine😊, Thank you Matt